packages/vector_graphics_compiler/lib/src/draw_command_builder.dart

```diff
@@ -14,8 +14,6 @@ class DrawCommandBuilder {
 
   /// Add a save layer to the command stack.
   void addSaveLayer(Paint paint) {
-    assert(paint.fill!.color != null);
```

Comment:
Calling saveLayer without a color on it is broken at runtime IIRC. Why are we removing this assert?

At the very least I think we'd need opaque black on here - is that added elsewhere?

Reply:
Fill colors default to opaque black and Fill.color is no longer nullable

Reply:
Ahhh ok.